apps/web/src/modules/job/appliedActor/components/ActorCardHeader/index.tsx

```diff
@@ -26,7 +31,16 @@ const ActorCardHeader = (
       <Typography variant="h6" sx={{ flexGrow: 1 }}>
         {firstName} {middleName} {lastName}
       </Typography>
-      <Chip {...CHIP_VARINTS[status!]} />
+      {props.jobStatus === JobStatus.SELECTION_ENDED ? (
+        <IconButton
+          href={`/job/${jobId}/actor/${actorId}/refund`}
+          onClick={(ev) => ev.stopPropagation()}
+        >
+          <ReportOutlined fontSize="medium" color="error" />
+        </IconButton>
```

Comment:
should we check that application is offer accepted (we shouldn't be able to refund rejected actor)

Reply:
and also check that this actor doesn't have pending refund request ?

Reply:
Should check offer accepted but for pending refund request might not possible in this PR because API does not implement this part yet